modules/thekla_unwrap/SCsub

```diff
@@ -60,7 +60,7 @@ if env['builtin_thekla_atlas']:
     if (not env.msvc):
         env_thekla_unwrap.Append(CXXFLAGS="-std=c++11")
 
-    if env["platform"] == 'x11':
+    if env["platform"] == 'native_unix':
```

Comment:
Is this an oversight? Shouldn't this also be generic_unix?

Reply:
Aye, another oversight. Thanks!